text/0005-application-formalization.md

```diff
@@ -0,0 +1,196 @@
+* Start Date: 10/09/2023
+* Contributors:
+  * [Craig McNally](cmcnally@ebsco.com)
+  * [Vince Bareau](vbareau@ebsco.com)
+* RFC PRs:
+  * PRELIMINARY REVIEW: https://github.com/folio-org/rfcs/pull/14
+  * DRAFT REFINEMENT: https://github.com/folio-org/rfcs/pull/22
+  * PUBLIC REVIEW: https://github.com/folio-org/rfcs/pull/29
+  * FINAL REVIEW: TBD
+* Outcome: <!-- Leave this blank. Will eventually be either ACCEPTED or REJECTED -->
+
+# Application Formalization
+
+## Summary
+It is proposed that an Application be defined as the minimal but complete set of elements which together are intended to deliver a specific solution to Folio. The intention is to group together all the closely related components which will need to evolve together, but independently of other parts (i.e. other applications) of Folio.
+
+## Motivation
+The existing Folio project lacks a formal definition of an Application structure. A consequence is that Folio is stuck in a mode of monolithic releases which are ever increasing in size.  There have also been repeated complaints that the setup and management of Folio can be difficult due to the number modules involved.
+
```

Comment:
How does the notion of an Application address the problem of monolithic releases? Would FOLIO release become smaller because this notion is introduced? Would some existing functionality be removed from FOLIO release and instead provided in form of an external Application? If so, which specific functionality would that be?